### PR TITLE
{Release} HotFix: Release 2.17.1

### PR DESCRIPTION
**Pull request by msyyc**
HotFix: Release 2.17.1

### CODE REVIEW

File: src/azure-cli-core/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.17.1
+++++++
+* Minor fixes
```
Comment (Juliehzl) [inline src/azure-cli-core/HISTORY.rst:8]: no change

Comment (msyyc) [reply src/azure-cli-core/HISTORY.rst:8]: 2.17.1
++++++
* No change

like this?

**Comment by yonzhan** [comment]
HotFix

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+2.17.1
+++++++
+
+**RDBMS**
+
+* Hotfix: `az mysql create`: fix for bad parameter name 'serv_name' to 'service_name'
```
Comment (jiasli) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* Hotfix: `az mysql create`: Revert bad parameter name 'serv_name' to 'service_name'
```

**Review by jiasli — APPROVED** [review]

**Comment by jiasli** [comment]
Please do remember to follow [Hotfix PR guideline](https://github.com/Azure/azure-cli/blob/9c68bfa839309acc92b5684700d758e900f9e198/doc/authoring_command_modules/README.md#hotfix-pr) to create another PR to merge `release` back to `dev` **locally**, like #15991.

**Pull request merged**